admin-auth/src/test/kotlin/com/epam/drill/admin/auth/CoffeineCacheServiceTest.kt

```diff
@@ -54,4 +54,10 @@ class CoffeineCacheServiceTest {
             assertNotNull(principal)
         }
 
+    @Test
+    fun `given principle with value null, then cache must not populate`(): Unit = runBlocking {
```

Comment:
Better to rename `given null principle in callback function, getFromCacheOrPutIfAbsent must return null and cache must not populate`

Reply:
renamed